lang/cs/Org.Apache.REEF.IMRU.Examples/OnREEFIMRURunTimeConfiguration.cs

```diff
@@ -72,9 +73,11 @@ internal static IConfiguration GetYarnIMRUConfiguration()
             IConfiguration imruClientConfig =
                 REEFIMRUClientConfiguration<TMapInput, TMapOutput, TResult>.ConfigurationModule.Build();
 
+            var fileSystemConfig = HadoopFileSystemConfiguration.ConfigurationModule.Build();
```

Comment:
This assumes that IMRU on REEF will always use this particular file system. While this is true in the here and now, it doesn't seem like a defensible position long term. Instead, we should requite the user to supply this configuration. Maybe {{IMRUJobDefinition}} needs a new field for {{DriverConfiguration}}?